Version 2.16 (#1512)
kadena-io/chainweb-node: CHANGELOG.md

# `chainweb-node` Changelog

## 2.16 (2022-08-23)

This version replaces all previous versions. Any prior version will stop working
on **2022-09-01T00:00:00Z**. Node administrators must upgrade to this version
before that date.

This version will expire on **2022-12-01T00:00:00Z**.

To upgrade, pull the latest docker image or download the binary and restart the
node.

Changes:

> *   Implement principal namespaces.

Author: ```suggestion
```

*   Upgrade to Pact 4.4. This release brings namespaced keysets. For further changes check the [Pact Changelog](https://github.com/kadena-io/pact/blob/master/CHANGELOG.md).

> *   Improve performance during catchup.
> *   Improve P2P protocol performance.
> *   Improve TLS session management.
> *   Faster and safer binary serialization across the board.

Author: ```suggestion
*   Improve performance during catchup. (?)
*   Improve P2P protocol performance. (#1502)
*   Improve TLS session management. (#1489)
*   Faster and safer binary serialization across the board. (#1494)
```

# `chainweb-node` Changelog

## 2.16 (2022-08-23)

This version replaces all previous versions. Any prior version will stop working
on **2022-09-01T00:00:00Z**. Node administrators must upgrade to this version
before that date.

This version will expire on **2022-12-01T00:00:00Z**.

To upgrade, pull the latest docker image or download the binary and restart the
node.

> Changes:

Author: @larskuhtz suggested that we include this link here: https://github.com/kadena-io/chainweb-node/compare/2.15...master

> Changes:

Author: ```suggestion
Changes (https://github.com/kadena-io/chainweb-node/compare/2.15...master):
```

> Changes:

Author: ```suggestion
Changes (https://github.com/kadena-io/chainweb-node/compare/2.15...2.16):
```

*   Implement principal namespaces.

> *   Upgrade to Pact 4.4. This release brings namespaced keysets, and new
>     zero-knowledge primitives.

Reviewer: ```suggestion
*   Upgrade to Pact 4.4. This release brings namespaced keysets. For further changes check the [Pact Changelog](https://github.com/kadena-io/pact/blob/master/CHANGELOG.md).
```

*   Improve performance during catchup.
*   Improve P2P protocol performance.
*   Improve TLS session management.
*   Faster and safer binary serialization across the board.

> *   Use rocksdb-specific features to improve performance.
> *   rocksdb is now built alongside chainweb-node instead of using the system
>     rocksdb installation.

Author: ```suggestion
```
